tests/behat/features/capabilities/groups/flexible/content/view/outsider-anonymous/groups-flexible-content-view-outsider-anonymous.feature

```diff
@@ -244,3 +244,37 @@ Feature: Flexible groups content view access for anonymous users
     When I am viewing the event "Test content"
 
     Then I should be asked to login
+
+  Scenario: As anonymous user views a group event on overview page
```

Comment:
Thanks a lot for the additional test coverage, that helps in the confidence especially for these kind of impactful changes 😍